src/component/button/HsiButton/HsiButton.tsx

```diff
@@ -181,17 +200,35 @@ export class HsiButton extends React.Component<HsiButtonProps, HsiButtonStatePro
             }
           );
 
-      featureInfoUrls.push(featureInfoUrl);
-
+      if (featureInfoUrl.indexOf('geoserver.action') >= 0) {
+        const ftName: string = layer.getSource().getParams().LAYERS;
```

Comment:
Will this gracefully handle different spellings of `LAYERS`, or are we sure it'll always be uppercase?

Reply:
When looking at the source i would say "yes"
https://github.com/openlayers/openlayers/blob/v6.4.3/src/ol/source/TileWMS.js#L254